### DISCUSSION TITLE
Question: How to injection css？

### DISCUSSION THREAD

**Discussion by getoutnow2013** [quoid/userscripts#718] [Q&A] [answered]
I've make 2 scripts to inject.

TestAlert.js - is a simply javascript alert
TestHideCss.js - inject hide everything under body

But only alert shows, inject css is fail.
![testUserScript1](https://github.com/user-attachments/assets/ac8a59fd-b48b-4bf8-9faf-db06cd63882e)

How to inject css success? Can anyone give me a sample?
Thanks.

TestAlert.js Code:
```
// ==UserScript==
// @name         TestAlert
// @description  Alert
// @author       Userscripts
// @version      0.0.1
// @match        *://*/*
// @grant        none
// @inject-into  content
// ==/UserScript==

(function () {
  alert("This is a test script");
})();
```

TestHideCss.js Code:
```
// ==UserScript==
// @name         TestHideCss
// @description  Test CSS
// @version      0.0.1
// @author       getoutnow
// @match        *://*/*
// @grant        GM_addStyle
// ==/UserScript==

(function () {
  const myStyleCss = `body.readable { display:none !important; }`;
  document.body.className += 'readable';
  GM_addStyle(myStyleCss);
})();
```

**Comment by ACTCD** [comment, marked as answer]
https://github.com/quoid/userscripts#api

Use `GM.addStyle(css)` instead of `GM_addStyle`.
